### ISSUE TITLE
[Chips] TextField chips should include placeholder in VoiceOver readout on selection

### ISSUE THREAD

**Issue by jmdetloff** [material-components/material-components-ios#4315]
Similarly to textfields, the placeholder should be read out by voice over on selection.

**Comment by andrewoverton** [comment]
Reopening due to b/110878735
